_drafts/2018-03-01-q2-goals.md

```diff
@@ -0,0 +1,77 @@
+
+### Cloud compatibility
+![Cloud compatibility](/img/posts/cloud_compatibility.jpg "sensenet goes to cloud")
+The next milestone in the lifetime of SN7 is cloud compatibility which involves lot of additional researches and developments. 
+
+Read below some challenges we will be working on in Q2:
+
+#### Azure SQL & AD
+You can already use Azure SQL to host sensenet’s database but the installer does not support Azure Active Directory authentication yet so you need to use SQL authentication during the install process. We know about this and we will fix this soon, so you'll be able to install to Azure without this workaround.
+
+#### Messaging and cache
+We will start working on refactoring messaging and caching in sensenet by examining the industry standard and implementation opportunities. During the first phase we’ll create a study from the current state and opportunities, then we can move on by forming needs and actions for future improvements to modernize messaging and caching. As cloud compatibility is already in the corner, sensenet messaging in cloud environment is also a hot topic in our product team.
+
+#### Disk usage
+Like messaging and caching, disk usage is a bit different in cloud environment than on premise that's why we will research how Azure operations affect our disk usage and see if any further modifications are needed in the product.
+
+
+### sensenet UX revamp
+![UX revamp](/img/posts/excavator.jpg "UX revamp")
+After so many under the hood changes like the componentization, as one of the biggest upgrade in SN7, the user interface will get an overhaul as well. 
+We want to create a UI which is future proof and amazig to use, so we started working with a UX researcher who are contacting our partners to collect as much feedback as he can.
+
+We would like to understand how people really use sensenet and make it more simple and easy to understand, so you’ll find every button, controls and other elements at the right place, you won’t need to serch for them.
+
+The core concept of this revamp is to introduce a new simplified user interface with only the required features and separate it from the content explorer.
+Users will be able to edit permissions, view and manage document but they wont have access to the content explorer (which will be also rewritten) and to more advanced options.
+
+Stay tuned because we’ll share more info about the new sensenet UX in the coming months.
+
+
+### DMS v2.0
+In Q2 we will publish the 2.0 version of our document management system which is started back in 2017 as a DMS MVP project. Now we want to go further by implementing the following features into the 2.0 version:
+- Copy and Move action
+- Improved path picker
+- Real time document editing
+
+Check out the current state of the DMS right [here](https://github.com/SenseNet/sn-dms-demo). 
+Feel free to clone and explore the solution. 
+
+### Dynamic Content Type editing
```

Comment:
This is only a discussion thread about custom metadata, I wouldn't mention this yet, especially not using 'near' and 'without coding'. I think we should remove this section too, it is too early to list this.